docs/paper.md

```diff
@@ -46,14 +46,14 @@ In the most basic setup, at least two executables call preCICE to perform a coup
 As additional software components, so-called adapters bridge the gap between the preCICE API and the software environments used by the coupled models.
 Creating and using this overall setup for early development purposes is not only cumbersome, but also very inefficient.
 The artificial solver testing environment (ASTE) allows to replace and imitate models coupled via preCICE by artificial ones, potentially in parallel distributed across multiple ranks on distributed memory.
-This helps in the development of preCICE, adapters, or simulation setups by reducing the necessary software components and simplifying execution workflows.
+This helps in the development of preCICE, adapters, or simulation setups by reducing the necessary software components, simplifying execution workflows, and reducing runtime of the case.
 In addition, ASTE provides performance and accuracy metrics of the configured simulation setup.
 
 # Statement of need
 
 \autoref{fig:dependency-graph} illustrates the software stack required for a coupled simulation setup using FEniCS and OpenFOAM as examples, and compares it to a simulation setup using ASTE.
 Beside preCICE itself, core ingredients for practical applications are preCICE API language bindings, preCICE adapters, the simulation frameworks and their dependencies.
-Note that the framework dependencies are usually heavy by themselves, e.g., packages such as PETSc or Trilinos.
+Note that the framework dependencies are usually heavy by themselves, e.g., packages such as Dune or Trilinos.
```

Comment:
I would void PETSc as it is a dependency of precice

Reply:
PETSc is optional though. I put it because it's well known and established. If we put Dune, I assume we would have to cite it?

Reply:
Citations for dune https://www.dune-project.org/publications/dune-references.bib

We could also include packages like HyPr, ScaLAPACK, [SUNDIALS](https://computing.llnl.gov/projects/sundials), Kokkos, Ginkgo, Pforest.

Reply:
```suggestion
```